web/deeamon128/html/something-new/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Astronomy</title>
+</head>
+<body>
+  <section class="container active">
+    <header>
+      <h1 class="logo">astro</h1>
+      <i class="logo-icon"><img src="assets/logo.png" alt=""></i>
+      <div class="toggle-menu active"></div>
+    </header>
+    
+    <video src="assets/1.mp4" muted autoplay loop></video>
+    
+    <div class="overlay"></div>
+    
+    <div class="text">
+      <h1>Sagan Ipsum</h1>
+      <p>"Apollonius of Perga extraplanetary. Tingling of the spine. Orion's sword, rich in heavy atoms cosmic ocean astonishment encyclopaedia galactica tesseract two ghostly white figures in coveralls and helmets are soflty dancing, rich in heavy atoms the only home we've ever known how far away."</p>
+      <a href="#">Explore</a>
+    </div>
+
+    <ul class="social">
+      <li><a href="#"><img src="assets/fb.png" alt=""></a></li>
+      <li><a href="#"><img src="assets/insta.png" alt=""></a></li>
+      <li><a href="#"><img src="assets/gitbhub.png" alt=""></a></li>
+    </ul>
+  </section>
+
+  <div class="menu">
+    <ul>
+      <li><a href="#">Home</a></li>
+      <li><a href="#">News</a></li>
+      <li><a href="#">Observing</a></li>
+      <li><a href="#">Community</a></li>
+      <li><a href="#">Subscribe</a></li>
+    </ul>
+  </div>
+
+  <script src="app.js"></script>
```

Comment:
Next time, please add the script tag in the <head> section, and defer it: 
<head>
   ...........
   <script src="app.js" defer></script>
 </head>
 
You can read more about deferring at https://www.w3schools.com/tags/att_script_defer.asp

Reply:
Thank you. I will try now to see the difference. I always thought I that have to put it in the <body> section.